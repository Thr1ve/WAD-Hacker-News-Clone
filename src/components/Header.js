import React from 'react';

const Header = () =>
  <div className="nav">
    <div className="nav-left">
      <div className="nav-item">Top</div>
      <div className="nav-item">Best</div>
      <div className="nav-item">New</div>
      <div className="nav-item">Ask</div>
      <div className="nav-item">Show</div>
      <div className="nav-item">Job</div>
    </div>
  </div>;

export default Header;
