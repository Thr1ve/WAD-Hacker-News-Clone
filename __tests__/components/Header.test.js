import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/components/Header';

describe('Header', () => {
  it('should render correctly', () => {
    const onClick = () => {};
    const createOnClick = () => onClick;
    const component = renderer.create(
      <Header createClickHandler={createOnClick}/>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
