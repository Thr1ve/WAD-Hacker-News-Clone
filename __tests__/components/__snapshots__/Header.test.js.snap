exports[`Header should render correctly 1`] = `
<div
  className="container">
  <div
    className="nav">
    <div
      className="nav-left">
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        Top
      </div>
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        Best
      </div>
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        New
      </div>
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        Ask
      </div>
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        Show
      </div>
      <div
        className="nav-item"
        onClick={[Function onClick]}>
        Job
      </div>
    </div>
  </div>
</div>
`;
